### DISCUSSION TITLE
Product attribute filter/export filter

### DISCUSSION THREAD

**Discussion by zssvaidar** [atrocore/atrocore#568] [Q&A] [answered]
When i try to use filters,
request to ```http://domain/api/v1/Attribute/action/filtersData```
result in Bad Request
I've aleady increased `max_execution_time` and `memory_limit`
I suspect attributeTypeValue i have stored in my DB is too large, one of attributes contains more than 40k attributeTypeValues
Could you guys provide some assistance solving this problem, I could provide my attribute table dump.
![image](https://user-images.githubusercontent.com/30953478/207786688-f688347b-d27f-4adc-9122-22e1c98f0aa8.png)

**Comment by rratsun** [comment]
We have created an internal issue for this. We will fix it soon.

**Reply by zssvaidar** [reply]
![image](https://user-images.githubusercontent.com/30953478/207827724-c4356cff-15a0-4b56-b1de-80319400225d.png)
Here is the actual problem, some limit of 1000 Mb failed to retrieve after

**Comment by zssvaidar** [comment]
When ommited this linked families everything worked, issue was when there are more than 3500+ linked attribute families were present php couldn't process it anymore 
![image](https://user-images.githubusercontent.com/30953478/208827192-b056ef19-ad97-4183-8d6b-3c66727f766a.png)

**Reply by atrolex** [reply]
After update is implemented the attribute filters will be selectable via right-side panel with a small list of all attributes. There you can filter attributes like a normal entity.

**Comment by rratsun** [comment, marked as answer]
Hello,

Refactored at PIM 1.6.5. Please, update your system.
